grouper/background.py

```diff
@@ -34,19 +35,20 @@ def capture_exception(self):
 
     def run(self):
         while True:
-            logging.debug("Sending emails...")
             try:
                 session = Session()
+                logging.debug("Sending emails...")
                 process_async_emails(self.settings, session, datetime.utcnow())
+                logging.debug("Expiring edges....")
+                expire_edges(session)
+                logging.debug("Pruning old traces....")
                 prune_old_traces(session)
```

Comment:
Let's make that a separate change if just because I think we want to restructure the exception handling so that we can log success and failure properly.  Right now, there isn't an exception block where we can clearly determine what failed.